test/e2e/framework/pod/wait.go

```diff
@@ -153,41 +155,31 @@ func WaitForPodsRunningReady(ctx context.Context, c clientset.Interface, ns stri
 			Pods:                   podList.Items,
 		}, nil
 	})).WithTimeout(timeout).Should(framework.MakeMatcher(func(s *state) (func() string, error) {
-		replicas, replicaOk := int32(0), int32(0)
```

Comment:
One of the more important changes is that I'm no longer bothering to inspect the status of the Replicas. The only thing this was really doing was potentially prevent the function from returning.

Reply:
I suspect that was present for a reason. Not checking this anymore might break the use case of the original author.